sdk/client.go

```diff
@@ -17,11 +17,12 @@ import (
 type Client interface {
 	Checker(ctx context.Context, check *health.CheckState) error
 	Health() *healthcheck.Client
-	PostJob(ctx context.Context, headers Headers) (*models.Job, error)
-	PatchJob(ctx context.Context, headers Headers, jobID string, body []PatchOperation) (*RespHeaders, error)
-	PostTask(ctx context.Context, headers Headers, jobID string, taskToCreate models.TaskToCreate) (*RespHeaders, *models.Task, error)
-	GetTask(ctx context.Context, headers Headers, jobID, taskName string) (*RespHeaders, *models.Task, error)
-	GetTasks(ctx context.Context, reqheader Headers, jobID string) (*RespHeaders, *models.Tasks, error)
+	PostJob(ctx context.Context, reqHeaders Headers) (*RespHeaders, *models.Job, error)
```

Comment:
I think only change for this PR is for PutJobNumberOfTasks method, pls avoid making any changes for other methods.

Reply:
Yes ok. It's just a little change because I noticed that the PostJob function was not returning the *RespHeaders but all the other functions do.